salt/cache/consul.py

```diff
@@ -81,8 +81,11 @@ def __virtual__():
             'verify': __opts__.get('consul.verify', True),
             }
 
-    global api
-    api = consul.Consul(**consul_kwargs)
+    try:
+        global api
+        api = consul.Consul(**consul_kwargs)
+    except AttributeError:
+        return (False, "Failed to invoke consul.Consul, please make sure you have python-consul >= 0.2.0 installed")
```

Comment:
@johnj Can you add this requirement to the docs near the top of this file for the lib requirement as well?

Just something like:
```
:depends: python-consul >= 0.2.0
```
That will help people trouble shoot this as well.

Reply:
Thanks @rallytime, incoming update 😄